mirage/views/wb.ts

```diff
@@ -0,0 +1,9 @@
+import { HandlerContext, Schema } from 'ember-cli-mirage';
+
+export function moveFile(this: HandlerContext, schema: Schema) {
+    const fileId = this.request.params.id;
+    const nodeId = this.request.requestBody.resource;
+    const file = schema.files.find(fileId);
+    file.attrs.userId = null;
+    file.attrs.nodeId = nodeId;
```

Comment:
Trying to figure out [the docs](http://www.ember-cli-mirage.com/docs/v0.4.x/models/#instance-methods), it seems like `file.update()` is the way to do this. It's not documented (afaict) that mutating `file.attrs` will persist the changes in mirage's db, and I'm hesitant to rely on that even if it happens to work.